### PR TITLE
Authshifting

### PR DESCRIPTION
**Pull request by s0ham075**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details:
-  moved auth api route to app router 
- did some changes according to the previous review

## Any Breaking changes:
- Some pages might break since we havent moved all the pages to app router , alhough i have tried running the app and its working correctly
- One test is still failing for "should navigate to the sponsor page"

## Associated Screenshots:

- None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "react-moveable": "^0.54.1",
     "react-textarea-autosize": "^8.5.3",
     "satori": "^0.10.4",
-    "sharp": "^0.32.6",
+    "sharp": "^0.32.5",
```
Comment (NiallJoeMaher) [inline package.json]: Is there a reason to downgrade this or just older code? 🤔

File: pages/index.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const Home: NextPage = () => {
   return (
     <>
       <Head>
-        <title>Codú - The Web Developer Community</title>
+        <title>Codú -  Web Developer Community</title>
```
Comment (NiallJoeMaher) [inline pages/index.tsx]: Could we revert this for the moment?

**Review by NiallJoeMaher — COMMENTED** [review]
Really good work! thanks

File: pages/index.tsx (hunk 2 above)
Comment (s0ham075) [reply pages/index.tsx]: This was by mistake, i think i fixed this in the last commit.

File: package.json (hunk 1 above)
Comment (s0ham075) [reply package.json]: This was by mistake too so I fixed that

**Comment by s0ham075** [comment]
Thanks ! Should I make another pr or would you resolve the conflicts yourself?

**Comment by NiallJoeMaher** [comment]
@s0ham075 if you could add a commit to this branch that would be great! There shouldn’t be much conflicts or that yet. 🦾

**Comment by s0ham075** [comment]
Hey I think it should be fixed now

File: package.json (hunk 1 above)
Comment (NiallJoeMaher) [reply package.json]: This still hasn't been installed back as the right version. If you need a hand let me know. 🦾

Comment (s0ham075) [reply package.json]: On it !

**Review by NiallJoeMaher — APPROVED** [review]
LGTM! Thanks 🦾 🌮

**Pull request merged**